### DISCUSSION TITLE
How to know my keboard layout

### DISCUSSION THREAD

**Discussion by mateus-sartorio** [hyprwm/Hyprland#7165] [Q&A] [answered]
Hello. I just installed hyprland, and it is my first time using something other than GNOME. My keyboard is PT-BR ABNT2 (from Brazil). I would like to change my keyboard layout, but I dont event know what is the identifier my keyboard type (for example, I am using the default config now, which is "us").

**Comment by MightyPlaza** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/9a225b2c-9334-4982-9b51-e9caba3574e0)
